Changes.md

```diff
@@ -15,6 +15,7 @@ Improvements
   - The enabled/disabled state of a cell may now be edited directly via a new switch in the popup editing window.
   - When promoting plugs - such as those from Attributes node - which have their own "enabled" switch, this switch is now adopted by the spreadsheet, instead of having an additional "enabled" switch on the cell.
   - Added support for creating columns from tweaks in ShaderTweaks nodes. This allows the mode and value to be grouped in a single column.
+  - Added colour swatches for columns containing a switch as well as a colour value.
```

Comment:
🎉 🎉 🎉 🎉 🎉 
Kind of makes me want to put the color swatch first, and make it square/larger in the standard plug value widget too...